emscripten.py

```diff
@@ -471,7 +471,7 @@ def make_emulated_param(i):
                                 'shiftLeftByScalar'];
     fundamentals = ['Math']
     if settings['USE_PTHREADS']:
-      fundamentals += ['SharedInt8Array', 'SharedInt16Array', 'SharedInt32Array', 'SharedUint8Array', 'SharedUint16Array', 'SharedUint32Array', 'SharedFloat32Array', 'SharedFloat64Array', 'Atomics']
+      fundamentals += ['I8Array', 'I16Array', 'I32Array', 'U8Array', 'U16Array', 'U32Array', 'F32Array', 'F64Array', 'Atomics']
```

Comment:
could we just use the same `I8Array` etc. in both threads and no threads mode? That would let us remove the if here, and the ones on lines 769 (around the `access_quote` stuff). Probably requires a change somewhere else for the non-threads case, but it should be tiny.